<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrder</class>
 <widget class="QDialog" name="AddOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加订单</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/myimages/icon2/账单_新增.png</normaloff>:/myimages/icon2/账单_新增.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>99</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>Hotel Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>Room Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hotelNameInput">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入酒店名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roomTypeBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>单人间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双人间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三人间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大床间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>套间</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>15</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>Order Information</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="beginDate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2018</year>
     <month>7</month>
     <day>30</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2018</year>
     <month>7</month>
     <day>30</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2018</year>
     <month>7</month>
     <day>30</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
   <property name="date">
    <date>
     <year>2018</year>
     <month>7</month>
     <day>30</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>Begin Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>End Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>Order Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
   </property>
   <property name="text">
    <string>¥</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddOrder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
